src/fight.cpp

```diff
@@ -5728,7 +5747,7 @@ void roll_individual_initiative(struct char_data *ch)
         && !MOB_FLAGGED(FIGHTING(ch), MOB_INANIMATE)
         && GET_MOBALERT(FIGHTING(ch)) == MALERT_CALM
         && success_test(GET_REA(ch), 4) > success_test(GET_REA(FIGHTING(ch)), 4)) {
-      GET_INIT_ROLL(FIGHTING(ch)) = 0;
+      GET_INIT_ROLL(FIGHTING(ch)) <= 0;
```

Comment:
Typo? This changed from an assignment to a no-op comparison.

Reply:
Oops, yeah, not sure how that happened.